website/addons/wiki/utils.py

```diff
@@ -154,3 +155,64 @@ def format_wiki_version(version, num_versions, allow_preview):
         raise InvalidVersionError
 
     return version
+
+def serialize_wiki_settings(user, node_ids):
```

Comment:
@sloria, I did some more testing and this does not seem to be a problem.

To return the desired information, `serialize_wiki_settings` needs to touch all the nodes at some point, and `include_wiki_settings` helps stop it from having to load information about chunks of components that are irrelevant. While it is recursive,  `get_descendants_recursive()` is a generator that is usually not exhausted.

That being said, there are definitely a better ways to write this function.
Should I rewrite it, or is it okay the way it is?